src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
```

Comment:
The class 'WorkWithFile' should have a private constructor to prevent instantiation if it's intended to be a utility class.